### DISCUSSION TITLE
Burger Component

### DISCUSSION THREAD

**Discussion by Todor46** [saadeghi/daisyui#2644] [Ideas / feature requests]
I want to contribute by adding a simple hamburger component for toggling other components such as drawer and similar. I believe it is a type of component that is used a lot and can benefit future users. 

If approved, I'll open a PR.

**Comment by saadeghi** [comment]
I see hamburger more as a icon, not a component.
Currently people can use any SVG icon for that. I'm not sure if turning an icon into a CSS class would add a value 🤔
